tfhe/src/core_crypto/gpu/vec.rs

```diff
@@ -68,30 +115,66 @@ impl<T: Numeric> CudaVec<T> {
     ///
     /// - `streams` __must__ be synchronized to guarantee computation has finished, and inputs must
     ///   not be dropped until streams is synchronised
-    pub unsafe fn memset_async(&mut self, value: T, streams: &CudaStreams)
+    pub unsafe fn from_cpu_multi_gpu_async(src: &[T], streams: &CudaStreams) -> Self {
```

Comment:
Why now we have a `_multi_gpu_` variant for memset and copy?

Reply:
The multi gpu variant does the memsest or copy on all GPUs, while with the single GPU one we can select on which GPU we want to copy/memset.